src/install.rst

```diff
@@ -106,7 +106,7 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
                      mamba create -n nextstrain \
                        -c conda-forge -c bioconda \
-                       nextstrain-cli augur auspice nextalign snakemake git \
+                       nextstrain-cli augur auspice nextalign snakemake git epiweeks pangolin pangolearn \
```

Comment:
_non-blocking_

Did you look into if adding these three packages significantly increases the installation time or size on disk?

Reply:
I hadn't, but that would be good to know! Here's some benchmarks:

### Installation time

Note that each `time` is taken after cleaning all caches so these are worst-case scenarios given my processor and network speed. Not much difference here (93.63s vs. 98.21s user+sys time).

```sh
conda clean --all --force-pkgs-dirs
time mamba create -n tmp1 \
    -c conda-forge -c bioconda \
    nextstrain-cli augur auspice nextalign snakemake git \
    --yes
# 52.33s user 41.30s system 92% cpu 1:41.63 total

conda clean --all --force-pkgs-dirs
time mamba create -n tmp2 \
    -c conda-forge -c bioconda \
    nextstrain-cli augur auspice nextalign snakemake git epiweeks pangolin pangolearn \
    --yes
# 55.48s user 42.73s system 90% cpu 1:48.14 total
```

### Disk usage

1.5G -> 1.6G (~0.1G increase).

```sh
du -hs /opt/homebrew/Caskroom/miniconda/base/envs/tmp1
# 1.5G	/opt/homebrew/Caskroom/miniconda/base/envs/tmp1
du -hs /opt/homebrew/Caskroom/miniconda/base/envs/tmp2
# 1.6G	/opt/homebrew/Caskroom/miniconda/base/envs/tmp2
```

Reply:
Thanks!